pkg/occlient/occlient.go

```diff
@@ -1672,7 +1672,8 @@ func (c *Client) StartBuild(name string) (string, error) {
 }
 
 // WaitForBuildToFinish block and waits for build to finish. Returns error if build failed or was canceled.
-func (c *Client) WaitForBuildToFinish(buildName string) error {
+func (c *Client) WaitForBuildToFinish(buildName string, stdout io.Writer) error {
+	following := false
```

Comment:
Need more description on this, bit confused why we're setting following to false then true when going through the channel loop.